### DISCUSSION TITLE
Best Offline Labelling Tool for Yolo5

### DISCUSSION THREAD

**Discussion by IssaIssa1** [ultralytics/yolov5#5687] [Q&A]
Good day, 

I would like to hear on your experience on the best tools for offline labelling for yolo5 model. I have found great tools for online labelling but it's requested to handle the data offline. 

I would like to hear your thoughts on the matter. 
Best

**Comment by glenn-jocher** [comment]
@IssaIssa1 I would recommend Roboflow for online labelling due to their easy export, but if your workflow requires offline labelling then you might try CVAT.
